addons/pos_restaurant/static/tests/tours/pos_restaurant.js

```diff
@@ -30,6 +31,36 @@ function isSyncStatusConnected() {
         },
     ];
 }
+function checkLastOrderPreparationChange(expected_changes) {
+    return [
+        {
+            content: `Check last order preparation changes with expected changes ${JSON.stringify(expected_changes)}`,
+            trigger: ".pos", // dummy trigger
+            run: function() {
+                const currentOrder = posmodel.get_order();
```

Comment:
can you try accessing `posmodel` from `window` i.e. `window.posmodel` ? This might fix the runbot issue